### DISCUSSION TITLE
Using SqlBuilder.SET

### DISCUSSION THREAD

**Discussion by skarm61** [maxtoroq/DbExtensions#36] [Q&A] [answered]
Does DbExtension support Mysql? I use it with mysql and got some issues when i update records using 

```
... .SET(field_name, value) ...
```

I get query like:

```
...
SET field_name, field_name
WHERE ...
```

except 

```
SET field_name = @p0, field_name = @p1
```

Can you help me?

**Comment by maxtoroq** [comment, marked as answer]
The first parameter of the SET method is a format string. That means you have to do this:

``` csharp
...
.SET("field_name = {0}", value)
```

If you are getting the field name from a variable then you can do this:

``` csharp
...
.SET(field_name + " = {0}", value)
```

**Comment by skarm61** [comment]
Thanks for answer. It was my fail. In previous project I used it like 

```
.SET("field_name = {0}", value)
```

And now, when i start to use properties and attributes to autogenerate update and insert queries, I made this mistake. I should use 

```
.SET(field_name + " = {0}", value)
```

But it could be cool if you add one more method SET, which will receive property and value.

**Comment by maxtoroq** [comment]
You can extend SqlBuilder using extension methods. My intention is to keep is simple and consistent.
